### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by Alexr16**
### In this. PR:
- Created a website consisting of 3 pages: Home, Calculator, and Quote.
- Styled using React basic styling (inline styles, importing CSS, SCSS).
- Used React-Router to route Home, Calculator, and Quote page.
- Added navigation links for all 3 routes displayed on each page.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -62,3 +55,30 @@ body {
...
+h1 {
+  font-size: 2em;
+  margin: 0;
+}
+
+.cal-cont {
+  display: flex;
+  justify-content: space-between;
+  margin: 1rem;
+}
```
Comment (Whoistolu) [inline src/App.css]: - [x] [OPTIONAL] Great implementation @Alexr16 , however, the UI could be made better by placing the currently at the center of the page by aligning and justifying it to the center.

### Your Implementation
Screenshot from 2022-04-26 21-56-58

File: src/pages/Home.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    </p>
+    <p>
+      Aliquam ornare eget velit id pharetra. Fusce efficitur aliquam ullamcorper. Donec
+      tristique, arcu vitae facilisis semper, tellus sem consequat augue, vel dictum nisl
+      nulla sed diam. Donec pellentesque orci a velit eleifend, suscipit laoreet nibh pretium.
+      Praesent ultricies sodales ligula quis sagittis. Vivamus vehicula molestie molestie. Sed
+      tincidunt eget neque sit amet consequat. Aenean turpis enim, gravida a euismod et,
+      feugiat at ante.
+    </p>
+  </div>
```
Comment (Whoistolu) [inline src/pages/Home.js:24]: - [ ] [OPTIONAL] Great job done @Alexr16 , however, kindly refer to my recommendation concerning the calculator and implement the same here. Once again, great job 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @Alexr16 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (Alexr16) [reply src/App.css]: @Whoistolu I'll work on that.
Thanks for your recommendation.

**Pull request merged**